reco_utils/recommender/deeprec/deeprec_utils.py

```diff
@@ -298,6 +329,10 @@ def create_hparams(flags):
         # dkn
         wordEmb_file=flags["wordEmb_file"] if "wordEmb_file" in flags else None,
```

Comment:
We can use `dict.get(key, None)` here

Reply:
I completely agree with you, this is more pythoninc, in fact, we could do directly `dict.get(key)` because the default is alternative is `None`. However, changing this piece of code would break a lot of the code on deeprec and I guess probably there will be a lot of refactoring needed. 

I would leave it as it is, unless @Leavingseason thinks we should change it

Reply:
Yes, let's leave this up to the original author.